Bio/Seq.py

```diff
@@ -1130,7 +1132,9 @@ class UnknownSeq(Seq):
     Seq('ACGT????')
     """
 
-    def __init__(self, length, alphabet=None, character="?"):
+    def __init__(
+        self, length: int, alphabet=None, character: Optional[str] = None,
+    ) -> None:
```

Comment:
Can you set the character default back to "?" here (I can see how this happened while resolving a merge conflict, is used to be be None when the alphabet was used).

Reply:
Oops, will do